src/bitcoin/d/mod.rs

```diff
@@ -1134,6 +1134,23 @@ impl BitcoinD {
         }
     }
 
+    /// Get mempool entry.
+    // TODO: Should existing calls to `is_in_mempool` use this function instead and check is_some()?
```

Comment:
Yes. :)

Reply:
That's done now. Do you think it should be called `get_mempool_entry()` or `mempool_entry()`, or perhaps a mixture of both depending on where the method is?

Reply:
I generally follow this rule: longer but explicit / clear is better than shorter but implicit / less clear. If clear already, shorter is better. In this case i think `mempool_entry()` is a better fit.